docs/source/fitting/fitfunctions/Bessel.rst

```diff
@@ -0,0 +1,40 @@
+.. _func-Bessel:
+
+==============================
+Bessel
+==============================
+
+.. index:: Bessel
+
+Description
+-----------
+
+.. math:: A(t)=A_0J_0(2\pi\nu t+\phi)
+
+where,
+
+:math:`A_0` is the amplitude,
+
+:math:`\nu` is the frequency of oscillation (MHz),
+
+:math:`\phi` is the phase at :math:`t=0`,
+
+and :math:`J_0(x)` is the Bessel Function of the first kind, its expression is given by:
+
+.. math:: J_0(x)=\sum_{l=0}^{\infty}\frac{(-1)^l}{2^{2l}(l!)^2}x^{2l}.
+
+.. figure:: /images/Bessel.png
+   :alt: Bessel.png
```

Comment:
`:alt:` should be a description (this is shown if the image fails to load or cannot be shown for whatever reason).